gems/sorbet-runtime/lib/types/helpers.rb

```diff
@@ -21,7 +21,7 @@ def final!
   end
 
   def sealed!
-    Kernel.raise "TODO(jez) sealed is not implemented in the runtime yet."
+    Private::Sealed.declare(self, Kernel.caller(1..1)&.first&.split(':')&.first)
```

Comment:
We compute the decl_file here to avoid having to make `Private::Sealed.declare` aware of how many stack frames deep it was under the call to `sealed!`. i.e., here the decl_file is the first thing in the caller stack. But inside `declare`, it's potentially arbitrary many things deep (though in practice it's the second).